Travel_fellowships.md

```diff
@@ -1,41 +1,183 @@
-# Open Bioinformatics Foundation Travel Fellowships
+# Open Bioinformatics Foundation Event Fellowships
 
-The Open Bioinformatics Foundation ([OBF](https://www.open-bio.org)) is pleased to announce a new OBF Travel Fellowship program aimed at increasing diverse participation at events promoting open source bioinformatics software development and open science in the biological research community.
+The Open Bioinformatics Foundation ([OBF](https://www.open-bio.org)) promoting open source bioinformatics software development and open science in the biological research community. 
+The OBF Event Fellowship program is aimed at increasing diverse participation at events promoting open science practices such as resource development and dissemination in the bioinformatics and biological research community.
```

Comment:
I would prefer to keep closer to the original wording with "open source bioinformatics" or at least "open source" in the text. This term does appear again later though.

Reply:
Thanks @peterjc. Can you please edit in your suggestions? I don't know where exactly this comment refers to.

Reply:
If the green shows suggested new text, there's a typo that needs to be fixed:
"The Open Bioinformatics Foundation ([OBF](https://www.open-bio.org)) promoting open source" - should be "promotes"